services/core/PlatformDriverAgent/HomeAssistantDriver.md

```diff
@@ -0,0 +1,239 @@
+**VOLTTRON Home Assistant Driver.** 
+
+Please see the README for the platform driver.
+[services/core/PlatformDriverAgent/README.md](https://github.com/riley206/Rileys_volttron/blob/55146b78d3ab7f53d08598df272cdda2d0aa8d3d/services/core/PlatformDriverAgent/README.md)
+```mermaid
+  sequenceDiagram
+    HomeAssistant Driver->>HomeAssistant: Retrieve Entity Data (REST API)
+    HomeAssistant-->>HomeAssistant Driver: Entity Data (Status Code: 200)
+    HomeAssistant Driver->>PlatformDriverAgent: Publish Entity Data
+    PlatformDriverAgent->>Controller Agent: Publish Entity Data
+
+    Controller Agent->>HomeAssistant Driver: Instruct to Turn Off Light
+    HomeAssistant Driver->>HomeAssistant: Send Turn Off Light Command (REST API)
+    HomeAssistant-->>HomeAssistant Driver: Command Acknowledgement (Status Code: 200)
+
+```
+The first thing you will need is the IP address and your long lived access token from your Home Assistant instance. 
```

Comment:
It might be better to use

1. Get the long lived access code from your home assistant instance.
    Instructions for doing that here....
2. Install a platform driver 
3. etc 
4.